test/01LessonOneTest.js

```diff
@@ -5,36 +5,37 @@ var assert = require('assert');
 
 function returnTrue() {
     // should return true
-
+return true;
 }
 
 function returnFalse() {
     // should return false
-
+    return false;
 }
 
 function greaterThan(big, small) {
     // should return true if the first argument is greater than the second argument
     // and return false if the first argument is less than the second argument
-
+    return big > small;
 }
 
 function lessThan(small, big) {
     // should return true if the first argument is less than the second argument
     // and return false if the first argument is greater than the second argument
-
+    return small < big;
 }
 
 function equalTo(a, b) {
     // should return true if the first argument is equal to the second argument
     // and should return true if the first argument is equal to the second argument
-
+    return a===b;
 }
 
 function notEqualTo(a, b) {
     // should return true if the first argument is not equal to the second argument
     // and should return false if the first argument is equal to the second argument
 
+    return a!==b;
 }
```

Comment:
@paulrenenichols, going well, I think. But I cannot figure how out to test my code in node (i.e., how can I act like a user and enter my word into a prompt to be converted to piglatin?). When I put everything into the Chrome console, it looks like it works, but I am not positive until I see it as the user. The only other thing I believe I am missing is the normalization. Just not sure where to put this  - at the beginning or at the end (inside the function, I know).